packages/tfchain_client/src/nodes.ts

```diff
@@ -21,6 +21,7 @@ export interface Node {
   virtualized: boolean;
   serialNumber: string;
   connectionPrice: number;
+  hasGpu: number;

```

Comment:
this does not exist on the node object on the chain, no?

Reply:
Not yet. I removed gpu changes & will implement them once added to the chain.